src/components/Table/expression.ts

```diff
@@ -0,0 +1,55 @@
+
+const getPartExpression = function (object: string, data: any) {
```

Comment:
@ankitsaraogi can we use arrow function here?

Reply:
@dheeraja00 : there is no need of arrow function as I dont want to pass this context inside function.